commit 891c28f657838f23bdbba35d1644f0ca78969385
Author: Valentin David <me@valentindavid.com>
Date:   Wed May 15 17:55:08 2024 +0200

    linux: Force DRM_I915 to be a module
    
    For some cards, DRM_I915 requires firmware. This is not correct to
    have it as built-in if firmware is not embedded in the kernel.
    Otherwise initialization fail, and DRM is not available on the device.

diff --git a/files/linux/fdsdk-config.sh b/files/linux/fdsdk-config.sh
index 2cc9af03a..04785c180 100644
--- a/files/linux/fdsdk-config.sh
+++ b/files/linux/fdsdk-config.sh
@@ -640,7 +640,7 @@ module SND_SOC
 module SND_SOC_AMD_ACP
 case "$arch" in
     x86_64|i686)
-        module DRM_I915
+        remove DRM_I915; module DRM_I915
         module DRM_GMA500
     ;;
     aarch64)
